BuildResidentialHPXML/measure.rb

```diff
@@ -1623,18 +1676,35 @@ def arguments(model) # rubocop:disable Lint/UnusedMethodArgument
     arg.setUnits('Btu/hr')
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('heating_system_2_heating_airflow', false)
+    arg.setDisplayName('Heating System 2: Heating Airflow')
+    arg.setDescription("The output heating airflow of the second heating system. If not provided, the OS-HPXML autosized default (see <a href='#{docs_base_url}#hpxml-heating-systems'>HPXML Heating Systems</a>) is used.")
+    arg.setUnits('CFM')
+    args << arg
+
     arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('heating_system_2_heating_autosizing_factor', false)
     arg.setDisplayName('Heating System 2: Heating Autosizing Factor')
     arg.setDescription('The scaling factor applied to the auto-sizing methodology. If not provided, 1.0 is used.')
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('heating_system_2_heating_autosizing_limit', false)
+    arg.setDisplayName('Heating System 2: Heating Autosizing Limit')
+    arg.setDescription('The scaling limit applied to the auto-sizing methodology. If not provided, 1.0 is used.')
+    args << arg
+
     arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('heating_system_2_fraction_heat_load_served', true)
     arg.setDisplayName('Heating System 2: Fraction Heat Load Served')
     arg.setDescription('The heat load served fraction of the second heating system. Ignored if this heating system serves as a backup system for a heat pump.')
     arg.setUnits('Frac')
     arg.setDefaultValue(0.25)
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('hvac_distribution_fan_watts_per_cfm', false)
+    arg.setDisplayName('HVAC Distribution: Blower Fan Efficiency')
+    arg.setDescription("The blower fan efficiency at maximum fan speed. Applies only to #{HPXML::HVACTypeFurnace} heating system, #{HPXML::HVACTypeCentralAirConditioner} and #{HPXML::HVACTypeMiniSplitAirConditioner} cooling systems, and #{HPXML::HVACTypeHeatPumpAirToAir}, #{HPXML::HVACTypeHeatPumpMiniSplit}, and #{HPXML::HVACTypeHeatPumpGroundToAir} heat pumps. If not provided, the OS-HPXML default (see <a href='#{docs_base_url}#hpxml-heating-systems'>HPXML Heating Systems</a>, <a href='#{docs_base_url}#hpxml-cooling-systems'>HPXML Cooling Systems</a>, <a href='#{docs_base_url}#hpxml-heat-pumps'>HPXML Heat Pumps</a>) is used.")
```

Comment:
For mini-splits, it will only apply to ducted systems. Also need to update the logic below to incorporate that.

Reply:
I'm confused why the docs say "FanPowerWattsPerCFM defaults to 0.07 W/cfm for ductless systems and 0.18 W/cfm for ducted systems." for MSHP then?

Reply:
I think because MSHPs can be ductless or ducted. For ductless MSHP, the default would 0.07 but ducted MSHP it would 0.18.

Reply:
Oh, good point... Yes, I was wrong, this does apply to ductless MSHPs too.

Reply:
Though that raises the question about the name convention. It's weird to call the argument "HVAC Distribution: Blower Fan Efficiency" and allow it to apply to a ductless MSHP, which doesn't have a distribution system.

Reply:
I think that was sort of why I (ultimately) went the route of defining airflow rates at the system level -- the arguments map to system level HPXML fields.

Reply:
Here's my proposal: call it "HVAC: Blower Fan Efficiency". The description will indicate that it applies to split (i.e., not packaged) HVAC system types. We can do the same for airflow ("HVAC: Blower Heating Airflow Rate" and "HVAC: Blower Cooling Airflow Rate"), and it will apply to the same HVAC types. @jonwinkler, does this make sense to you?

Reply:
@shorowit, yes, that makes sense to me. I think these inputs are most useful for ducted systems.

Reply:
(Or I guess it could be "HVAC Blower: Fan Efficiency", "HVAC Blower: Heating Airflow Rate", etc.)